### ISSUE TITLE
pip does not install resources

### ISSUE THREAD

**Issue by kevinkovalchik** [ursgal/ursgal#152]
Hello,

I'm using Python 3.8.5 in Ubuntu 20.04.

The paths to the downloaded zip files when running `UController.download_resources` don't seem to be resolving properly. At some point it is using a relative path, but it is either being improperly defined or there is a chdir missing. When running, it fails on the first downloaded resource:

```
>>> ursgal.UController().download_resources()
Executable for msamanda_2_0_0_9706 is not available on your system
Engine msamanda_2_0_0_9706 cannot be downloaded automatically, please download the engine manually and move it to the appropriate folder
Executable for omssa_2_1_9 is not available on your system
[ - HTTP - ] Downloading files from https://www.sas.upenn.edu/~sschulze/ursgal_resources/ursgal/resources/linux/64bit/omssa_2_1_9/omssa_2_1_9.md ...
[ - HTTP - ] Saved file as /Data/Development/ursgal_test/venv/lib/python3.8/site-packages/ursgal/resources/linux/64bit/omssa_2_1_9/omssa_2_1_9.md
Remove tmp file: /Data/Development/ursgal_test/venv/lib/python3.8/site-packages/ursgal/resources/linux/64bit/omssa_2_1_9/omssa_2_1_9.md
MD5 check successful! The executables will be downloaded from https://www.sas.upenn.edu/~sschulze/ursgal_resources/ursgal/resources/linux/64bit/omssa_2_1_9/omssa_2_1_9.zip
[ - HTTP - ] Downloading files from https://www.sas.upenn.edu/~sschulze/ursgal_resources/ursgal/resources/linux/64bit/omssa_2_1_9/omssa_2_1_9.zip ...
[ - HTTP - ] Saved file as /Data/Development/ursgal_test/venv/lib/python3.8/site-packages/ursgal/resources/linux/64bit/omssa_2_1_9/omssa_2_1_9.zip
[   md5    ] Calculating md5 for omssa_2_1_9.zip ....
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/Data/Development/ursgal_test/venv/lib/python3.8/site-packages/ursgal/ucontroller.py", line 2791, in download_resources
    calculated_zip_md5 = self.calc_md5( zip_file_name )
  File "/Data/Development/ursgal_test/venv/lib/python3.8/site-packages/ursgal/unode.py", line 205, in calc_md5
    with open(input_file, mode='rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'ursgal/resources/linux/64bit/omssa_2_1_9/omssa_2_1_9.zip'
```

Note that the file does indeed exist when I manually look for it. When comparing the file it downloaded with the file it is looking for, the only way it would ever find it is if the working directory is the python environment's 'site-packages' directory (because the path used is relative):

```
[ - HTTP - ] Saved file as /Data/Development/ursgal_test/venv/lib/python3.8/site-packages/ursgal/resources/linux/64bit/omssa_2_1_9/omssa_2_1_9.zip
```
```
FileNotFoundError: [Errno 2] No such file or directory: 'ursgal/resources/linux/64bit/omssa_2_1_9/omssa_2_1_9.zip'
```

If I chdir into the environment's `site-packages` directory before calling `UController().download_resources()` then everything runs fine.

I'd also mention that the `download_resources` function doesn't seem to be mentioned in the Introduction section of the documentation. I see that it gets used to automatically download things when using `setup.py`, but when installing via `pip` it doesn't get invoked so you have to do it manually, which took a bit of digging to figure out as there is no mention that you need to do it yourself.

Best wishes,
Kevin

**Comment by StSchulze** [comment]
Hi Kevin,

Sorry to hear that this is not yet working as it should.
We had changed the download_resources fairly recently to be integrated in setup.py and, in theory, pip as well. But it sounds like there are still some hiccups for that. We will have a look into this, thanks for reporting the issue!

Best,
Stefan

**Comment by kevinkovalchik** [comment]
Hi Stephan,

Sounds good! Let me know if you need any more information.

By the way, I haven't actually had a chance to try the library yet, but I
really like the concept of it! It would have saved me a lot of time in
building a large-scale pipeline last year had I known about it...

Best wishes,
Kevin

**Comment by StSchulze** [comment]
Hi Kevin,

We modified the install procedure now and are about to merge it into the master #163 

Could you try the following:
`pip install ursgal==0.6.8rc3`
followed by
`ursgal-install-resources` (from anywhere on the system)
That should install Ursgal (and the third party tools that we are allowed to distribute) into your site packages.

Best,
Stefan

**Comment by StSchulze** [comment]
Solved in version 0.6.8.
--> Closed
